CHANGES.md

```diff
@@ -1,5 +1,20 @@
 # Changes
 
+## edge-21.12.4
+
+This release adds support for custom HTTP meathods in the viz stats
+(i.e CLI and Dashboard). Additionally, It also includes various
+other smaller improvements.
+
+* Added support for custom HTTP methods in the `linkerd-viz` stats
+* Updated the health checker to pull trust root from the `linkerd-identity-trust-roots`
+  configmap to support cases where they are generated externally (thanks @wim-de-groot)
+* Removed unnecessary `installNamespace` bool flag from the
+  `linkerd-control-plane` chart (thanks @mikutas)
+* Updated the `install` command to error if container runtime check fails
+* Added support for non-pod endpoints in `GetProfile` responses
```

Comment:
We could perhaps take this out? Not sure how user facing this change is, it's more of an implementation detail 🤔